web/geosearch/datapunt_geosearch/config.py

```diff
@@ -131,3 +134,29 @@ def get_db_settings(db: str, docker_host: str, localport: str) -> Dict[str, str]
     'uitgevoerdonderzoek',
     'bominslag'
 ]
+
+
+def load_jwks_config(filename):
+
+    with open(filename, 'r') as config_file:
+        try:
+            config = yaml.load(config_file)
+        except (yaml.YAMLError,
+                json.decoder.JSONDecodeError) as error_details:
+            logger.error(
+                "Failed to load config: %s Error: %s",
+                filename,
+                repr(error_details)
+            )
+            return None, None, None
+
+    return config.get('jwks'), config.get('jwks_url'), config.get('jwks_allowed_signing_algorithms')
+
+
+default_config_file = os.path.join(os.path.dirname(__file__), 'config.yaml')
+JWKS, JWKS_URL, JWKS_SIGNING_ALGORITHMS = load_jwks_config(default_config_file)
```

Comment:
This is actually an if-else: if `/etc/geosearch.yaml` is found, this code is completely ignored.

Reply:
This is absolutely correct, as `/etc/geosearch.yaml` is environment specific config and should overwrite default, if exists.